tests/phpunit/tests/test-links-multi-domains-to-one.php

```diff
@@ -42,8 +47,28 @@ public function tear_down() {
 	/**
 	 * @ticket #1296
 	 * @see https://github.com/polylang/polylang/issues/1296.
+	 *
+	 * @param bool $cache_languages Value of the constant `PLL_CACHE_LANGUAGES`.
+	 * @param bool $cache_home_url  Value of the constant `PLL_CACHE_HOME_URL`.
+	 *
+	 * @testWith [true, false]
+	 *           [false, true]
+	 *           [false, false]
 	 */
-	public function test_home_and_search_urls() {
+	public function test_home_and_search_urls( $cache_languages, $cache_home_url ) {
+		Functions\when( 'pll_get_constant' )->alias(
+			function ( $constant_name ) use ( $cache_languages, $cache_home_url ) {
```

Comment:
Is it the `use` keyword that prevent you from avoid code duplication?

Reply:
Are you suggesting to to put this mock in a trait?

Reply:
If it's doable, I think we should. Or even in the testcase.

Reply:
Yes, I think it's a good idea.
Done in [a549d3c](https://github.com/polylang/polylang/pull/1324/commits/a549d3c6e01a50a75a9a866df7ac332d09a2dfda).